packages/frontend-core/src/fetch/UserFetch.js

```diff
@@ -47,10 +47,17 @@ export default class UserFetch extends DataFetch {
         limit,
       }
       const res = await this.API.searchUsers(opts)
+      let tenantInfo
+      try {
+        tenantInfo = await this.API.getTenantInfo({ tenantId })
```

Comment:
NAB: would it make more sense to return an empty response from the API if there's no tenantInfo for the tenant, rather than just relying on it failing?